rel/i18n/emqx_authn_mnesia_schema.hocon

```diff
@@ -9,4 +9,14 @@ user_id_type.desc:
 user_id_type.label:
 """Authentication ID Type"""
 
+bootstrap_file.desc:
+"""The bootstrap file imports users into the built-in database."""
```

Comment:
```suggestion
"""The bootstrap file imports users into the built-in database. Note that bootstrap is only performed when the built-in database table is empty."""
```

Reply:
Not realy.
That's how the original api_key's bootstrap_file was designed. It only imported when it was empty, and then after many iterations, it became imported every time it was started. So now this will be consistent with api_key's as well.